Sources/Commands/Workspace.swift

```diff
@@ -216,6 +236,54 @@ public class Workspace {
         }
     }
 
+    /// Find the managed dependencies for the given package name.
+    ///
+    /// It is possible that the provided package name is correct but
+    /// dependency is not yet fetched. This method will throw an error
+    /// in that case.
+    ///
+    /// - Throws: WorkspaceOperationError
+    func findDependency(package: String) throws -> ManagedDependency {
+        // Load the external manifests.
+        let (_, manifests) = try loadDependencyManifests()
+        // Find the dependency.
+        guard let url = manifests.first(where: { $0.name == package })?.url,
+              let dependency = dependencyMap[RepositorySpecifier(url: url)] else {
+            throw WorkspaceOperationError.dependencyNotFound(package)
```

Comment:
I'm not sure why, conceptually this is just querying a collection for membership, and that would be an optional in Swift.

BTW, one other issue with this function is embedding the loadDependencyManifests is pretty expensive. It might be better to make `loadDependencyManifests` return a structured object, and then put this method on that. That way clients that want to find a bunch of dependencies aren't wasting work. That change might also make other parts of Workspace clearer.